src/memory_manager.rs

```diff
@@ -197,6 +233,28 @@ pub fn alloc_slow<VM: VMBinding>(
     mutator.alloc_slow(size, align, offset, semantics)
 }
 
+/// Invoke the allocation slow path. This function is NOT a GC safepoint. If MMTk fails
+/// to allocate memory, it will not attempt a GC, nor call [`crate::vm::Collection::out_of_memory`],
+/// MMTk instead return a zero address. See [`alloc_no_gc`] for more details.
+///
+/// Notes on [`alloc_slow_no_gc`] also apply to this function.
```

Comment:
I rephrased it a bit and refererence `alloc_no_gc`.

Instead of saying "notes on xxx also apply", I references `alloc_slow` and briefly quotes the related note.

```suggestion
/// Invoke the allocation slow path of [`alloc_no_gc`].
///
/// Like [`alloc_no_gc`], this function is guaranteed not to trigger GC and not to call
/// `Collection::block_for_gc` or [`crate::vm::Collection::out_of_memory`].  It returns zero on
/// allocation failure.
///
/// Like [`alloc_slow`], this function is also only intended for use when a binding implements the
/// fastpath on the binding side.
```